src/ember-app/config/environment.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-env node */
+'use strict';
+
+module.exports = function(environment) {
+  let ENV = {
+    modulePrefix: 'ember-app',
+    environment,
+		rootElement: '#ember-app',
```

Comment:
Actually, that should go in an APP hash inside of ENV (see further down in the test section where it is configured to run Ember in testing mode)